ui-tests/src/test/java/org/openmrs/reference/page/AdministrationPage.java

```diff
@@ -20,7 +20,7 @@
     private final static By MANAGE_VISIT_TYPES = By.cssSelector("#legacyui-manageVisitTypes a");
     private static final By MANAGE_PROVIDERS = By.cssSelector("#content a[href='/openmrs/admin/provider/index.htm']");
     private static final By MANAGE_PERSONS = By.cssSelector("#content a[href=\"/openmrs/admin/person/index.htm\"]");
-
+    private static final By MANAGE_ROLES = By.cssSelector("#legacyui-manageRoles a");
```

Comment:
```suggestion
    private static final By MANAGE_ROLES = By.cssSelector("#legacyui-manageRoles a");
    
```

Reply:
This xss verifies the already fixed xss vulnerability , so i was trying to do this to track exactly where the xss vulnerability occurs

Reply:
Resolved thanks